src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+function parseSalary(salaryStr) {
+  return Number(salaryStr.replace(/[$,]/g, ''));
+}
+
+function sortList(list) {
+  const itemsArray = Array.from(list.children);
+
+  itemsArray.sort(
+    (a, b) => parseSalary(b.dataset.salary) - parseSalary(a.dataset.salary),
```

Comment:
The sorting logic seems to be reversed. The current comparator function sorts the list in descending order by salary, but if the requirement is to sort in ascending order, you should subtract `parseSalary(a.dataset.salary)` from `parseSalary(b.dataset.salary)` instead.